Tests/test_file_icns.py

```diff
@@ -82,6 +82,23 @@ def test_jp2_icon(self):
             self.assertEqual(im2.mode, 'RGBA')
             self.assertEqual(im2.size, (wr, hr))
 
+    def test_getimage(self):
+        with open(TEST_FILE, 'rb') as fp:
+            icns_file = IcnsImagePlugin.IcnsFile(fp)
+
+            im = icns_file.getimage()
+            self.assertEqual(im.mode, "RGBA")
+            self.assertEqual(im.size, (1024, 1024))
+
+            im = icns_file.getimage((512, 512))
+            self.assertEqual(im.mode, "RGBA")
+            self.assertEqual(im.size, (512, 512))
+
+    def test_not_an_icns_file(self):
+        with open('Tests/images/invalid.icns', 'rb') as fp:
+            self.assertRaises(SyntaxError,
+                lambda: IcnsImagePlugin.IcnsFile(fp))
```

Comment:
assertRaises could be used without lambdas:

```python
self.assertRaises(SyntaxError, IcnsImagePlugin.IcnsFile, fp)
```

I find such format cleaner.

Reply:
Okay, I've added a commit.